packages/custom-functions-metadata-plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "custom-functions-metadata-plugin",
-  "version": "1.6.2",
```

Comment:
@millerds This version update should be done by Lerna, right? I don't think the version should be manually updated, but cannot quite remember.

Reply:
Correct . . . do not update the version, that will be done as part of the publish process.

Reply:
Reverted.